replay_testing/decorators/fixtures.py

```diff
@@ -49,5 +49,5 @@ def __call__(self, cls):
         return cls
 
     @staticmethod
-    def parameterize(fixture_list: list[McapFixture]):
+    def parameterize(fixture_list: list):
```

Comment:
Why untype this?

Reply:
Marked this more specifically now, as `list[McapFixture | BaseFixture]`

Really `BaseFixture` is actually `BaseRemoteFixture`, because there isn't an abstract fixture base class.